Lib/sqlite3/test/backup.py

```diff
@@ -0,0 +1,127 @@
+import sqlite3 as sqlite
+from tempfile import NamedTemporaryFile
+import unittest
+
+class BackupTests(unittest.TestCase):
```

Comment:
Please add tests to make sure that keyword-only parameters are really keyword-only:

```py
self.cx.backup(bckfn.name, 1, lambda x, y: None)  # should raise TypeError
```